robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentingClassLoader.java

```diff
@@ -1184,4 +1150,107 @@ private ClassReader typeInfo(final String type) throws IOException {
       }
     }
   }
+
+  /**
+   * GeneratorAdapter implementation specific to generate code for Robolectric purposes
+   */
+  private static class RobolectricGeneratorAdapter extends GeneratorAdapter {
+    private final boolean isStatic;
+    private final String desc;
+
+    public RobolectricGeneratorAdapter(MethodNode methodNode) {
+      super(Opcodes.ASM4, methodNode, methodNode.access, methodNode.name, methodNode.desc);
+      this.isStatic = Modifier.isStatic(methodNode.access);
+      this.desc = methodNode.desc;
+    }
+
+    public void loadThisOrNull() {
+      if (isStatic) {
+        loadNull();
+      } else {
+        loadThis();
+      }
+    }
+
+    public boolean getIsStatic() {
```

Comment:
The purpose was to change it to a verb, following http://www.oracle.com/technetwork/java/codeconventions-150003.pdf